homeassistant/components/plex/media_player.py

```diff
@@ -90,49 +90,53 @@ def _async_add_entities(
 class PlexMediaPlayer(MediaPlayerEntity):
     """Representation of a Plex device."""
 
+    available = False
+    name = None
+    state = STATE_IDLE
+
+    media_content_type = None
+    media_content_id = None
+    media_duration = None
+    media_image_url = None
```

Comment:
`media_image_url` is a property in the base media_player entity. Please keep it a property.